core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.spec.ts

```diff
@@ -249,10 +249,16 @@ describe('Topic editor tab directive', function() {
     expect($scope.topicDescriptionChanged).toEqual(true);
   });
 
-  it('should call the TopicUpdateService if name is updated', function() {
+  fit('should call the TopicUpdateService if name is updated', function() {
```

Comment:
did you force push this PR?

Reply:
oh yeah it needed some changes..ill be converting it to a draft for some time